client/state/domains/management/actions.js

```diff
@@ -132,9 +131,15 @@ export function saveWhois( domain, whoisData, transferLock ) {
 			domain,
 		} );
 
-		return wpcom
-			.undocumented()
-			.updateWhois( domain, whoisData, transferLock )
+		return wpcom.req
+			.post( {
+				path: `/domains/${ domain }/whois`,
+				apiVersion: '1.1',
+				body: {
+					whois: whoisData,
+					transfer_lock: transferLock,
+				},
+			} )
```

Comment:
Let's use the `.post( path, body )` form here, too.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/57537/commits/9e37b9f20abe1894fb8b7ddd06c9c172608b95b0.